client/ayon_nuke/plugins/create/create_write_render.py

```diff
@@ -58,20 +38,22 @@ def create_instance_node(self, product_name, instance_data):
         write_data.update(instance_data)
 
         # get width and height
-        if self.selected_node:
+        if node_selection:  # contains 1 Write node or nothing
```

Comment:
The selection seems to be broken now. Try it @robin-ynput following way:
1. select read node
2. open creator
3. chose Render (write)
4. keep Use selection activated
5. Create
Notice that popup will with empty window will tell you Wtite node....

Reply:
Thanks a lot, yep I realized I changed the previous behavior. 
I have fixed that.